blog/2024-08-14-conda-ecosystem-explained.mdx

```diff
@@ -0,0 +1,116 @@
+
+Furthermore, all of these tools use the same conda standards and formats, which are collaboratively improved through the [Conda Enhancement Proposal (CEP)](https://github.com/conda/ceps/blob/main/cep-1.md) process.
+
+## Conda Package Repository and Channels
+
+When it comes to managing packages with `conda` or conda-compatible tools, understanding where these packages come from is key. [Anaconda.org](https://anaconda.org/) serves as a central hub for conda packages, organized into various channels. While anyone can create their own channel on anaconda.org to host conda packages or even run their own package index, we'll focus on the largest and most significant channels within the conda ecosystem.
+
+### `conda-forge`
+
+[conda-forge](https://conda-forge.org) is one of the largest and most impactful channels in the conda ecosystem. Entirely community-driven, this channel hosts over 25,000 packages, making it a go-to resource for developers across the globe. conda-forge is open-source, free to use, and maintained by a group of dedicated volunteers. The [governance model](https://conda-forge.org/community/governance/) ensures that the channel remains reliable and accessible to everyone.
+
+### `bioconda`
+
+If your work involves computational biology, [Bioconda](https://bioconda.github.io) is the channel you’ll want to explore. Unlike conda-forge, which covers a wide variety of packages, Bioconda is focused on bioinformatics. It offers over 8,000 packages that cater to subdomains like genomics, proteomics, structural biology, and biomass analysis. Like conda-forge, bioconda is open-source and free to use, maintained by a community of volunteers.
+
+### `defaults`
+
+The `defaults` channel (sometimes referred to as `main`) is maintained by [Anaconda, Inc](https://www.anaconda.com/). It's important to note that this channel comes with [Terms of Service](https://legal.anaconda.com/policies/en/). While largely free for individual users, students, and small companies, there are scenarios where a license might be required. For more information, feel free to review [frequently asked questions about Anaconda’s Terms of Service](https://www.anaconda.com/pricing/terms-of-service-faqs).
+
+## Getting Started with conda and conda-compatible tools
+
+There are a few different ways to install and get started with conda (at the time of writing this article):
+
+<table>
+  <thead>
+    <tr>
+      <th>Installer name</th>
+      <th>Package manager included</th>
+      <th>Automatically sets channel as</th>
+      <th>Status</th>
```

Comment:
Not sure if this column is necessary.

Reply:
In what way, unnecessary to highlight that pixi is still emerging?